11_dictionary/dictionary.rb

```diff
@@ -0,0 +1,30 @@
+class Dictionary
+  def entries
+    @entries ||= {}
+  end
+
+  def add(hash)
+    if hash.is_a? Hash
+      entries.merge!(hash)
+    else # input is a String
```

Comment:
Please see first 2 unchecked items in https://trello.com/c/4q1bQyxN/8-free-time

Reply:
Refactored.